### PR TITLE
Blog post 2024 08 27

### PR DESCRIPTION
**Pull request by yamdeeyoumar**
How can I preview the post on our website?

Is there a better way to embed images?

Can I embed imgur images with their links?

Thank you for patience guys, initial draft

### CODE REVIEW

**Comment by yamdeeyoumar** [comment]
Do I add myself to author.yml or someone else has to do it?

**Comment by AugustDG** [comment]
> Do I add myself to author.yml or someone else has to do it?

You can add yourself!

**Comment by yamdeeyoumar** [comment]
It is looking for .js files but in the directories the corresponding files are .min.js

**Comment by AugustDG** [comment]
There's no way to preview unfortunately 😅 

As for embedding the images, yes! I changed it a bit and it works quite nicely :)

File: _posts/2024-08-27-actuator-accessibility.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+**Finally, using a drill, it was tested at high speed and the aforementioned torque:**
+
+<iframe width="560" height="315" src="https://youtube.com/embed/z4qgHHI9vMk" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" referrerpolicy="strict-origin-when-cross-origin" allowfullscreen></iframe>
+
+
+After about an hour of intermittently cycling with the drill, the 3D printed discs started wearing at the profile, forming ABS powder and a lot of backlash is observed in the drive.
+
+## Conclusion
+
+Further explorations into more accessible alternatives to the dual-stage timing belt reduction will also be conducted in the near future, check back for bi-weekly updates. :)
```
Comment (AugustDG) [inline _posts/2024-08-27-actuator-accessibility.md:106]: If you want @yamdeeyoumar, you could add a small conclusion about what you learned :)

**Review by AugustDG — APPROVED** [review]
Did some editing/formatting, but it look great! :D

**Review by AugustDG — APPROVED** [review]

**Review by AugustDG — APPROVED** [review]
LGTM!

**Comment by Shamiivan** [comment]
this closes #17

**Pull request merged**